rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -201,10 +206,19 @@ def get_airdrop_data(name: str, data_dir: Path) -> Tuple[Iterator, TextIO]:
             request = requests.get(url=AIRDROPS[name][0], timeout=DEFAULT_TIMEOUT_TUPLE)
         except requests.exceptions.RequestException as e:
             raise RemoteError(f'Airdrops Gist request failed due to {str(e)}') from e
-
-        with open(filename, 'w') as f:
-            f.write(request.content.decode('utf-8'))
-
+        try:
+            content = request.content.decode('utf-8')
+            # Here 20900 is the size of the smallest CSV file we track
+            if not csv.Sniffer().has_header(content) or len(request.content) < 20900:
```

Comment:
no need to put a comment here. Add `20900` to a constant defined at the top with a proper name and comment can go away.

example `SMALLEST_AIRDROP_SIZE`

Also csv.Sniffer() is probably fine for this, but may have false positives/negatives for unknown CSVs. As long as we test the airdrops should be fne.